src/Apps/Collect/Routes/Collection/Components/Header/DefaultHeader.tsx

```diff
@@ -36,6 +36,8 @@ export const fitHeaderArtworks = (
         return headerArtworks
       }
 
+      if (!artwork.node.image) continue
```

Comment:
🤔This is pretty foreign syntax for most folks. Can it be expressed more conventionally?

Reply:
Lol, I can convert this while to a trampoline or recursive function or something. I also literally never use loops, but this is what was there.

Reply:
Ahh, i see, i didn't realized we were within a loop 👍 All good.